perllib/FixMyStreet/App/Controller/Admin/Config.pm

```diff
@@ -0,0 +1,83 @@
+package FixMyStreet::App::Controller::Admin::Config;
+use Moose;
+use namespace::autoclean;
+
+BEGIN { extends 'Catalyst::Controller'; }
+
+=head1 NAME
+
+FixMyStreet::App::Controller::Admin::Config - Catalyst Controller
+
+=head1 DESCRIPTION
+
+Admin pages for viewing/editing site configuration
+
+=head1 METHODS
+
+=cut
+
+=head2 index
+
+This admin page displays the overall configuration for the site.
+
+=cut
+
+sub index : Path( '' ) : Args(0) {
```

Comment:
Why doesn't the index route seem to need the `page_error_403_access_denied` bit like the methods below?